examples/Cargo.toml

```diff
@@ -32,6 +32,7 @@ embedded-storage    = "0.3.1"
 esp-alloc           = { path = "../esp-alloc" }
 esp-backtrace       = { path = "../esp-backtrace", features = ["exception-handler", "panic-handler", "println"] }
 esp-hal             = { path = "../esp-hal", features = ["log"] }
+esp-hal-buzzer      = { path = "../esp-hal-buzzer", optional = true }
```

Comment:
I don't think I find this integral enough to be included in `examples`. Also I think xtasks is prepared to run examples that are placed elsewhere, e.g. in the `esp-hal-buzzer` crate.

Reply:
I added it there because `esp-hal-smartled` also has an example there. That being said, I can move the extra crates into a special `extras/` to not complicate the root with helper crates that are not part of the core HAL.